Article.sublime-snippet

```diff
@@ -0,0 +1,15 @@
+<snippet>
+   <description>AsciiDoc Article Start</description>
+   <content><![CDATA[${1:Title}
+${1/./=/g}
+$TM_FULLNAME <$TM_EMAIL>
+v1.0, $TM_YEAR
+:copyright: (c) $TM_ORGANIZATION_NAME, $TM_YEAR. All rights Reserved.
+:backend: ${2:html5|wordpress|slidy}
```

Comment:
`${2:html5|wordpress|slidy}` simply gives me `html5|wordpress|slidy`, is this intended?